### PR TITLE
fix: avoid reloading the page after changing network

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: avoid reloading the portal after users change the network on EVM wallets
* fix: store the changed account on browser (localstorage) after users change the address on EVM wallets 

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/hooks/custom-signature/useEvmAccount.ts [hunk 1]
```diff
@@ -63,14 +63,14 @@ export function useEvmAccount() {
 
       // subscribe to changes
       provider.on('accountsChanged', handleAccountsChanged);
-      provider.on('chainChanged', handleChainChanged);
+      // provider.on('chainChanged', handleChainChanged);
```
Comment (bobo-k2) [inline src/hooks/custom-signature/useEvmAccount.ts:66]: Remove commented code

File: src/v2/services/implementations/AssetsService.ts [hunk 2]
```diff
@@ -58,6 +59,11 @@ export class AssetsService implements IAssetsService {
     const provider = getEvmProvider(this.currentWallet as any);
     const web3 = new Web3(provider as any);
 
+    const resultCheckProvider = await handleCheckProviderChainId(provider);
+    if (!resultCheckProvider) {
+      throw Error('Please connect to the correct network in your wallet');
```
Comment (bobo-k2) [inline src/v2/services/implementations/AssetsService.ts:64]: use i18

**Review by bobo-k2 — CHANGES_REQUESTED** [review]

File: src/v2/services/implementations/AssetsService.ts (hunk 2 above)
Comment (impelcrypto) [reply src/v2/services/implementations/AssetsService.ts:64]: @bobo-k2 This is service file. Are you suggesting me to use the `useI18` from out side of the service file?

Comment (impelcrypto) [reply src/v2/services/implementations/AssetsService.ts]: Actually, I have found a way to use i18 in service and repository file ([ref](https://github.com/AstarNetwork/astar-apps/pull/1233/commits/f84a7e237b1a7c3f4d75e5ccf23a6c506e3b488e))!!

image

**Review by bobo-k2 — APPROVED** [review]

**Pull request merged**